osu.Game/Screens/Multi/Multiplayer/Multiplayer.cs

```diff
@@ -28,46 +28,46 @@ public override void OnResuming(IScreen last)
 
         protected override void UpdatePollingRate(bool isIdle)
         {
-            var timeshiftManager = (RealtimeRoomManager)RoomManager;
+            var playlistsManager = (MultiplayerRoomManager)RoomManager;
```

Comment:
Variable name doesn't match the type. (It was borked before, but now that we're fixing...)